include/ceed/jit-source/cuda/cuda-ref-operator-assemble-diagonal.h

```diff
@@ -10,7 +10,6 @@
 //------------------------------------------------------------------------------
 // Diagonal assembly kernels
 //------------------------------------------------------------------------------
-
```

Comment:
This enum probably shouldn't be here. Let me make a quick PR to fix

Reply:
Thanks, indeed it looks like I missed editing the same duplicated comments in `hip-ref-operator-assemble-diagonal.h` anyway.

Reply:
Change merged to main